junit-runtime/src/main/scala/com/novocode/junit/OutputCapture.scala

```diff
@@ -0,0 +1,47 @@
+package com.novocode.junit
+
+import java.io.ByteArrayOutputStream
+import java.io.PrintStream
+
+object OutputCapture {
+  def start(): OutputCapture = new OutputCapture()
+
+  private def getScalaOut(): PrintStream = {
+    try {
+      scala.Console.out
+    } catch {
+      case t: Throwable => null
+    }
+  }
+
+  private def setScalaOut(p: PrintStream): Boolean = {
+    try {
+      scala.Console.setOut(p)
+      true
+    } catch {
+      case t: Throwable => false
+    }
+  }
+}
+
+final class OutputCapture private[OutputCapture]() {
+  private val originalOut = System.out
+  private val originalScalaOut = OutputCapture.getScalaOut()
+  private val buffer = new ByteArrayOutputStream()
+  private val prBuffer = new PrintStream(buffer, true)
+  private val scalaOutSet: Boolean = OutputCapture.setScalaOut(prBuffer)
```

Comment:
Is this really what JUnit does? Seems strange... If JUnit doesn't handle Scala's out specially, neither should we...